src/Turn.js

```diff
@@ -1,8 +1,26 @@
+const Card = require('../src/Card');
```

Comment:
Wasn't working. Kept getting undefined when I was sure I had it down correctly. Then I realized I'd forgotten this very vital bit of code.